src/main/java/com/williamfiset/algorithms/sorting/RadixSort.java

```diff
@@ -1,8 +1,44 @@
-/** NOTE: This file is still under development */
 package com.williamfiset.algorithms.sorting;
 
```

Comment:
Can you add a description of the algorithm, how it works, what type of data you want to sort with it and some time complexity informaiton?